src/tox/session/commands/show_config.py

```diff
@@ -65,16 +64,15 @@ def version_info(parser):
     while to_visit:
         current = to_visit.pop()
         current_dist = importlib_metadata.distribution(current)
-        current_name = canonicalize_name(current_dist.metadata["name"])
```

Comment:
hmmm, I was pretty sure I added the canonicalize name because there were duplicates in the output

Reply:
Yeah, not sure what happened here but canonicalize name now makes it break... do you still have your use case to validate?